docs/components/Header.js

```diff
@@ -36,26 +72,84 @@ function Header() {
-                <Box dangerouslySetInlineStyle={{ __style: { marginBottom: '1px' } }}>
+                <Box
+                  display="none"
+                  mdDisplay="block"
+                  dangerouslySetInlineStyle={{ __style: { marginBottom: '1px' } }}
+                >
                   Gestalt Design System
                 </Box>
               </Flex>
             </Box>
           </Link>
         </Text>
-        <Text size="sm" weight="bold">
-          v{GestaltPackageJson.version}
-        </Text>
+        <Box paddingX={2}>
+          <IconButton
+            accessibilityControls="site-settings-dropdown"
+            accessibilityExpanded={isSettingsDropdownOpen}
+            accessibilityHaspopup
+            accessibilityLabel="Site settings"
+            icon="filter"
+            iconColor="darkGray"
+            onClick={() => setSettingsDropdownOpen((prevVal) => !prevVal)}
+            ref={anchorRef}
+            selected={isSettingsDropdownOpen}
+            size="sm"
+            tooltip={{ 'text': 'Site settings', 'zIndex': POPOVER_ZINDEX }}
+          />
+        </Box>
+        {isSettingsDropdownOpen && (
+          <Dropdown
+            anchor={anchorRef.current}
+            id="site-settings-dropdown"
+            onDismiss={() => setSettingsDropdownOpen(false)}
+            zIndex={POPOVER_ZINDEX}
+            isWithinFixedContainer
+          >
+            <Dropdown.Item
+              onSelect={() => onChangeColorScheme()}
+              option={{ value: 'isDarkMode', label: 'Custom link 1' }}
+            >
+              <Flex alignItems="center" justifyContent="between" flex="grow" gap={8}>
+                <Label htmlFor="darkMode-switch">
+                  <Text weight="bold">Dark mode</Text>
+                </Label>
+                <Switch
+                  switched={colorScheme === 'dark'}
+                  onChange={() => onChangeColorScheme()}
+                  id="darkMode-switch"
+                />
+              </Flex>
+            </Dropdown.Item>
+            <Dropdown.Item
+              onSelect={() => onTextDirectionChange()}
+              option={{ value: 'isRTL', label: 'Custom link 1' }}
+            >
+              <Flex alignItems="center" justifyContent="between" flex="grow" gap={8}>
+                <Label htmlFor="rtl-switch">
+                  <Text weight="bold">Right-to-left</Text>
+                </Label>
+                <Switch
+                  switched={textDirection === 'rtl'}
+                  onChange={() => onTextDirectionChange()}
+                  id="rtl-switch"
+                />
+              </Flex>
+            </Dropdown.Item>
+          </Dropdown>
+        )}
       </Box>
 
       {/* Spacer element */}
       <Box flex="grow" />
 
       <Box alignItems="center" display="flex" flex="shrink" marginStart={2} mdMarginStart={0}>
-        <DocSearch />
+        <Box paddingX={2}>
+          <DocSearch popoverZIndex={POPOVER_ZINDEX} />
```

Comment:
Instead of passing the z-index as a prop, can it be defined outside of this component and imported where needed?